CI/freebsd/01_install_dependencies.sh

```diff
@@ -24,7 +24,11 @@ install_obs-deps() {
     status "Install OBS dependencies"
     trap "caught_error 'install_obs-deps'" ERR
 
-    sudo pkg install -U -y $@
+    if [ -z "${DISABLE_PIPEWIRE}" ]; then
+	sudo pkg install -U -y $@ pipewire
+    else
+	sudo pkg install -U -y $@
+    fi
```

Comment:
When running this script by itself, it is impossible to set `DISABLE_PIPEWIRE` despite the usage message. You need to allow this script to set this value. See the `linux/01_install_dependencies.sh` scripts for an example.

Reply:
Somehow forgot to copy that from Linux config. Done.